packages/block-library/src/cover/edit.native.js

```diff
@@ -328,11 +372,16 @@ const Cover = ( {
 		</TouchableWithoutFeedback>
 	);
 
-	if ( ! hasBackground ) {
+	if ( ! hasBackground || isCustomColorPickerShowing ) {
 		return (
 			<View>
+				{ colorPickerControls }
```

Comment:
If we add here `isCustomColorPickerShowing` we could render these controls only when the picker is opened. The settings button _will_ show up but behind the modal. If you hide the modal without selecting a color it will disappear. We'd need to check with Thomas but this would allow us not to modify the `InspectorControls` logic. What do you think?